packages/core/src/queries/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { sql, IdentifierSqlTokenType } from 'slonik';
+
+import pool from '@/database/pool';
+
+export const totalRowCount = async (table: IdentifierSqlTokenType) =>
+  pool.one<{ count: number }>(sql`
+    select count(*)
```

Comment:
nit: `select count(*) as count`, but maybe slonik already did this.

Reply:
yep slonik did do the part. the returned data type is { count: 4 }